xml/art_solution_automation.xml

```diff
@@ -0,0 +1,435 @@
+zypper install --repo ha-clustering sapnwbootstrap-formula</screen>
+    </example>
+
+    <itemizedlist>
+      <title>Best Practises</title>
+      <listitem>
+        <para><link xlink:href="https://documentation.suse.com/sbp/all/html/SAP_HA740_SetupGuide_AWS/index.html"
+          >&netweaver; &ha; Cluster 7.40 for the AWS Cloud - Setup Guide (Code 12)</link></para>
+      </listitem>
+      <listitem>
+        <para><link xlink:href="https://www.suse.com/media/white-paper/sap_netweaver_availability_cluster_740_setup_guide.pdf"
+          >&netweaver; &ha; Cluster 7.40 - Setup Guide (Code 12)</link></para>
+      </listitem>
+      <listitem>
+        <para><link xlink:href="https://www.suse.com/media/white-paper/sap_netweaver_availability_cluster_740_setup_guide.pdf"
+          >Simple Stack - &netweaver; &ha; on &sle; 12 (SP1 or newer)</link></para>
+      </listitem>
+      <listitem>
+        <para><link xlink:href="https://documentation.suse.com/sbp/all/html/SAP_S4HA10_SetupGuide-SLE12/index.html"
+          >&s4h; - Enqueue Replication 2 &ha; Cluster - Setup Guide</link></para>
+      </listitem>
+      <listitem>
+        <para><link xlink:href="https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/high-availability-guide-suse"
+          >Azure</link></para>
+      </listitem>
+    </itemizedlist>
+  </sect1>
+
+  <sect1 xml:id="sec-sol-autom-ha-cluster-config">
+    <title>&ha; Cluster Automated Configuration</title>
+    <para>
+      &salt; formula <package>habootstrap-formula</package> for bootstrapping and managing
+      a high availability cluster using &crmshell;.
+    </para>
+    <formalpara>
+      <title>Link:</title>
+      <para>See <link xlink:href="https://github.com/SUSE/habootstrap-formula"/>.</para>
+    </formalpara>
+    <itemizedlist>
+      <title>Properties for <package>habootstrap-formula</package></title>
+      <listitem>
+        <para>
+          Run on premises and on Cloud service providers (Azure, GCP, AWS).
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Import all private GPG keys in all the nodes (as in the current automated deployment).
+          This might require to copy and paste the keys.
+        </para>
+      </listitem>
+    </itemizedlist>
+  </sect1>
+
+  <sect1 xml:id="sec-sol-autom-drbd-nfs-config">
+    <title>DRBD/NFS Automated Configuration</title>
+    <para>
+      &salt; formula <package>drbd-formula</package> for setup and managing a DRBD cluster
+      using the command <command>drbdadm</command>.
+    </para>
+    <remark>toms 2020-06-29: Isn't the following addition "dangerous" for SLE users?</remark>
+    <example>
+      <title>Installation</title>
+      <screen>zypper addrepo --refresh obs://network:ha-clustering:sap-deployments:devel ha-clustering
+zypper install --repo ha-clustering drbd-formula</screen>
+    </example>
+  </sect1>
+
+  <sect1 xml:id="sec-sol-autom-more">
+    <title>For More Information</title>
```

Comment:
There are also links within the SAP HANA Automated Deployment section - what's the difference to these? Are the first ones specific to SAP HANA Automated Deployment and those here are meant for all formulas? It would be helpful to have one sentence here what the links relate to.

Reply:
The links listed in the "SAP HANA Automated Deployment" section are specific to this section, the others in the "For More Information" section are general.

I've added a short sentence in the "For More Information" section.